Cython/Build/Cythonize.py

```diff
@@ -46,9 +46,8 @@ def find_package_base(path):
     return base_dir, package_path
 
 
-def cython_compile(path_pattern, options):
+def cython_compile(all_paths, options):
     pool = None
-    all_paths = map(os.path.abspath, extended_iglob(path_pattern))
```

Comment:
Note that this isn't just the `cythonize` CLI command but also a programmatic interface to Cython. I don't think it's worth breaking backwards compatibility for this. Having the `cython_compile()` function accept and evaluate glob patterns doesn't sound wrong to me.

Reply:
To be honest I did not expect this function to be public API. I have added commit respecting this API.